tensorflow/python/ops/nn_ops.py

```diff
@@ -3008,12 +3008,15 @@ def max_pool_with_argmax_v2(input,
                             padding,
                             data_format="NHWC",
                             output_dtype=dtypes.int64,
-                            name=None):
```

Comment:
This is the TF v2 version of this API, so we can add the new argument before the name argument instead of after, as the TF V2 API isn't stable yet.